src/test/compile-fail/lint-exceeding-bitshifts.rs

```diff
@@ -11,7 +11,7 @@
 #![deny(exceeding_bitshifts)]
 #![allow(unused_variables)]
 #![allow(dead_code)]
-#![feature(num_bits_bytes)]
+#![feature(num_bits_bytes, const_indexing)]
```

Comment:
(because I thought we had decided that such errors in such contexts should not cause compilation errors, at least not ones that cannot be controlled via lint-style allow/warn/deny switches, but instead should just generate code to panic at runtime?)